lib/unpackDEF.js

```diff
@@ -42,6 +52,12 @@ function unpackDEF(buffer, options = {}) {
 
    options.palette = palette
 
+   if (options.paletteOverwrite) {
+      for (let i = 0; i < options.paletteOverwrite.colors.length; i++) {
+         options.palette[options.paletteOverwrite.offset + i] = options.paletteOverwrite.colors[i]
+      }
+   }
```

Comment:
Instead of this, now that `.palette` is a function, simply do:
```js
// let palette = buffer.loop(...)
palette = options.palette(palette)
// ...
// options.palette = palette
```